include/wx/convauto.h

```diff
@@ -91,6 +93,14 @@ class WXDLLIMPEXP_BASE wxConvAuto : public wxMBConv
         return m_bomType;
     }
 
+    wxFontEncoding GetEncoding() const;
+
+    // Return true if the fall-back encoding is used
+    bool IsFallbackEncoding() const
```

Comment:
This name seems a bit confusing to me, I'd rather call it `IsUsingFallbackEncoding()` or `UsesFallbackEncoding()`.

I also wonder if it's really that useful -- couldn't the application code do `GetEncoding() == GetFallbackEncoding()` if it really needs to know?

Reply:
If's useful for objects constructed with custom fallback encoding -- otherwise it would need something like GetBOM() == wxBOM_None && !IsUTF8(). I'll rename it.